src/modules/economy/types.ts

```diff
@@ -0,0 +1,75 @@
+import { UserNode } from 'modules/users/types'
+import { RelayEdges } from 'types/graphql'
+
+// General typing
+export type SociSessionType = 'stilletime' | 'krysseliste' | 'societeten'
+
+export type TransactionHistoryType =
+  | ProductOrderNode
+  | TransferNode
+  | DepositNode
```

Comment:
Should probably be merged into a custom Transaction type which unifies them on the backend